content/learning-paths/smartphones-and-mobile/android_webgpu_dawn/1-webgpu-fundamentals.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Introduction to WebGPU
+weight: 2
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## What is WebGPU?
+
+WebGPU is the successor to [WebGL](https://www.khronos.org/webgl/wiki/), a well adopted modern API standard for interfacing with GPUs. WebGPU provides better compatibility with modern GPUs, support for general-purpose GPU computations, faster operations, and access to more advanced GPU features. It is designed to provide a _unifed access_ to GPUs, agnostic to GPU vendors and Operating System the application runs on. WebGPU is essentially a Render Hardware Interface built on top of various backends APIs like Vulkan, DirectX, Metal depending on OS/platform. This duplicated development effort is made once by the web browsers and made available to us through the webgpu.h header they provide.
+
+![WebGPU High Level View](images/webgpu_highlevel.png "WebGPU High Level View")
+
+## Why WebGPU?
+
+WebGPU has been designed taking into learnings from older standards like WebGL, OpenGL etc. and is the only graphics API that benefits from
+
+* Reasonable level of abstraction
+* Good performance
+* Cross-platform
+* Backed by W3C Standards group
+* Future-proof
+
+WebGPU is a standard and not actual API, so the implementation can be adopted and developed as an interface between native applications developed in any programming language and GPUs. Moreover the requirements in terms of performance for web pages is actually the same as for native application.
+
+{{% notice Note %}}
+When designing an API for the Web, the two key constraints are portability and privacy. We benefit here from the effort developed for portability, and fortunately the limitations of the API due to privacy considerations can be disabled when using WebGPU as a native API
+{{% /notice %}}
+
+## Why C++?
+
+Initial target for WebGPU was JavaScript and most of the initial implementation was done in JavaScript and the `webgpu.h` header file is written in C. In this learning path, we choose C++ because:
```

Comment:
"The initial target for WebGPU was JavaScript so most of the initial implementation was completed in this language. The webgpu.h header file used in many implementations is written in C.

In this learning path we choose C++ because:"

Reply:
Thanks, updated